tests/e2e/vmlistener_test.go

```diff
@@ -59,8 +59,10 @@ func (s *VMListenerTestParams) SetUpTest(c *C) {
 }
 
 func (s *VMListenerTestParams) TearDownTest(c *C) {
```

Comment:
Should be TearDownSuite() for the whole test and TearDownTest for a specific test, in gocheck.